yarn-project/aztec-nr/compressed-string/src/compressed_string.nr

```diff
@@ -0,0 +1,137 @@
+use dep::aztec::types::type_serialization::TypeSerializationInterface;
+use dep::protocol_types::utils::field::field_from_bytes;
+use dep::std;
+
+struct FCS{
```

Comment:
What does FCS stand for?

Reply:
Oh, will add a comment. "FixedCompressedString", just a fixed size compressed string to easier work with the storage.